main/dapps/dapp-templates.md

```diff
@@ -0,0 +1,56 @@
+## Dapp Templates
+
+When creating a new dapp using `agoric init`, you have the option of
+starting from a number of templates. To use a template other than the default, add the
+`--dapp-template` option:
+
+```sh
+agoric init --dapp-template dapp-fungible-faucet my-fungible-faucet
+```
+
+### Fungible Faucet Dapp
+
+[Github:
+dapp-fungible-faucet](https://github.com/Agoric/dapp-fungible-faucet#fungible-faucet-dapp)
+
+Gives fungible tokens on request. Frontend is raw JavaScript with
```

Comment:
I'd suggest adding a little information about what brand the tokens are or how to specify a brand for them, whatever's appropriate. Just "tokens" is extremely broad with respect for what it's doing.

Reply:
I think the broadness is appropriate in this case. The tokens are called "Token". They have no other meaning. :)

Reply:
OK.